job-descriptions/ux-designer.md

```diff
@@ -0,0 +1,78 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# UX Designer (San Francisco or remote)
+
+### About us
+
+Sourcegraph's mission is to enable every software developer to create products using the best technology. The innovations of the future will all rely on software, and we want to bring the future sooner by building the best tools that help developers answer questions about their code. You can learn more about our mission by reading [our master plan](https://sourcegraph.com/plan).
+
+Our values:
+
+- **Inclusivity**. To build a product that serves the needs of all developers, we need to have a diverse team. Sourcegraph is an equal opportunity workplace; we embrace diversity and welcome people from all backgrounds and communities.
+- **Openness**. We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company): our code, our product roadmap, and our company processes are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+- **Progress**. We make continuous progress on our product by [releasing a new version of Sourcegraph every month](https://docs.sourcegraph.com/dev/releases). Check out [our product roadmap](https://docs.sourcegraph.com/dev/roadmap) to see what we are working on next. We also hold [regular retrospectives](https://docs.sourcegraph.com/dev/retrospectives) to learn how we can work better as a team.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+We are searching for a creative, self-motivated UX designer who can iteratively improve the cohesive user expereience, while bringing clean and thoughtful UI to their designs. It is important that this person deeply understands developers and can craft an experience that delights a technical audience.
+
+#### Your responsibilities
+
+You will:
+
+- Contribute to overall strategy and decision-making about Sourcegraph’s features.
+- Drive continual progress towards a cohesive, delightful experience for a technical audience through iterative improvements.
+- Design elegant flows, wireframes, prototypes, sketches, and high-fidelity visuals.
+- Use prototyping and user-testing to inform and validate designs.
+- Help measure, learn, and build new and existing experiences.
+- Collaborate with product and engineers to bring ideas to life, and support the team to make incremental progress toward design excellence.
+- Leverage and evolve the style guide to create a clean, usable design system.
+
+We will encourage and support you to:
+
+- Collaborate directly with customers to collect feature requirements and inform product design.
+- Publish blog posts and give conference talks about your work at Sourcegraph.
+
+#### Your qualifications
```

Comment:
This feels like a lot of qualifications. Is it possible to trim this list down into something more concise without losing important fidelity?

In any case, make sure that this list is stack ranked by importance (most important at top).